.env.ci

```diff
@@ -1,6 +1,6 @@
 APP_NAME=publiq-platform
 APP_ENV=local
-APP_KEY=
+APP_KEY=base64:5f6ivEq1QCl8ylgkFEzMpU6npqDi5tGYRuG1LYzilb4=
```

Comment:
This is not needed because the CI pipeline contains this https://github.com/cultuurnet/publiq-platform/blob/main/.github/workflows/ci.yml#L69 for every step